src/software/ai/passing/pass.cpp

```diff
@@ -1,12 +1,10 @@
 #include "software/ai/passing/pass.h"
 
 
-Pass::Pass(Point passer_point, Point receiver_point, double pass_speed_m_per_s,
-           Timestamp pass_start_time)
+Pass::Pass(Point receiver_point, double pass_speed_m_per_s)
     : receiver_point(receiver_point),
-      passer_point(passer_point),
       pass_speed_m_per_s(pass_speed_m_per_s),
-      pass_start_time(pass_start_time)
+      pass_array({receiver_point.x(), receiver_point.y(), pass_speed_m_per_s})
```

Comment:
looks like the instance variables store the same information, can we make do with one so we don't have to worry about the two version being mismatched

Reply:
good pointed, removed the pass array